branch_tags/models.py

```diff
@@ -6,7 +6,7 @@
 
 class UserTag(models.Model):
     name         = models.CharField(max_length=45, unique=True)
-    user         = models.ManyToManyField(User, related_name='users')
+    user         = models.ManyToManyField(User, related_name='utags')
```

Comment:
ManyToManyField를 사용하실 때는 column을 복수(users)로 만들어 주셔야 합니다.
그래야 user 목록을 불러올 때 아래와 같이 가독성 있는 코드를 작성할 수 있습니다.
```python
user_tag = UserTag.objects.get(id=1)
'''
users = user_tag.users.all()
```
추가로 related_name은 가독성을 위해 줄여서 적지 말아주세요!
```suggestion
    users         = models.ManyToManyField(User, related_name='usertags')
```